### PR TITLE
Modify MainApp for AddressBook to FoodWhere

### LINKED ISSUE #254: Modify AddressBook to FoodWhere in Logs

### PR DESCRIPTION
**Pull request by Hikoya**
Resolves #254

### CODE REVIEW

File: src/main/java/foodwhere/model/ModelManager.java [hunk 1]
```diff
@@ -27,12 +27,12 @@ public class ModelManager implements Model {
...
 
     /**
-     * Initializes a ModelManager with the given addressBook and userPrefs.
+     * Initializes a ModelManager with the given addressBook in FoodWhere and userPrefs.
      */
     public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) {
         requireAllNonNull(addressBook, userPrefs);
 
-        logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
+        logger.fine("Initializing with stall list: " + addressBook + " and user prefs " + userPrefs);
```
Comment (clarence-chew) [inline src/main/java/foodwhere/model/ModelManager.java:35]: Perhaps "Initializing with data: " may be more accurate?

File: src/main/java/foodwhere/storage/JsonSerializableAddressBook.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
 import foodwhere.model.stall.Stall;
 
 /**
- * An Immutable AddressBook that is serializable to JSON format.
+ * An Immutable AddressBook in FoodWhere that is serializable to JSON format.
```
Comment (clarence-chew) [inline src/main/java/foodwhere/storage/JsonSerializableAddressBook.java:17]: Perhaps Immutable should be uncapitalized "immutable"?

**Review by clarence-chew — APPROVED** [review]
I have some concerns, but okay with merging.

File: src/main/java/foodwhere/MainApp.java [hunk 3]
```diff
@@ -79,14 +79,14 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         try {
             addressBookOptional = storage.readAddressBook();
             if (!addressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample AddressBook");
+                logger.info("Data file not found. Will be starting with a sample stall and review list");
             }
             initialData = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
         } catch (DataConversionException e) {
-            logger.warning("Data file not in the correct format. Will be starting with an empty AddressBook");
+            logger.warning("Data file not in the correct format. Will be starting with an empty stall list");
```
[the remark this answers is not in the document]
Comment (Hikoya) [reply src/main/java/foodwhere/MainApp.java]: Resolved in latest commit

File: src/main/java/foodwhere/storage/JsonSerializableAddressBook.java (hunk 2 above)
Comment (Hikoya) [reply src/main/java/foodwhere/storage/JsonSerializableAddressBook.java]: Resolved in latest commit

File: src/main/java/foodwhere/model/ModelManager.java (hunk 1 above)
Comment (Hikoya) [reply src/main/java/foodwhere/model/ModelManager.java]: Resolved in latest commit

**Review by MinHeinA — APPROVED** [review]
Has resolved comments raised by @clarence-chew . LGTM!

**Pull request merged**